### DISCUSSION TITLE
Curso Fullstack - Módulo Entity Framework - Aula Mapeamento de Categoria

### DISCUSSION THREAD

**Discussion by oliveiramdaniel** [balta-io/forum#601] [Tira Dúvidas] [answered]
Gostaria de saber o motivo que não consigo montar o mesmo codigo que foi demonstrado na aula:

builder.HasKey(x:Category => x.Id);

![1](https://github.com/balta-io/forum/assets/104657435/c4f229e4-388a-4686-a3b3-e8d675ad2081)

Abaixo o código apresentado na aula.

![image](https://github.com/balta-io/forum/assets/104657435/6c22403b-493c-47f9-b80b-32c2e0234365)

Obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @oliveiramdaniel, como vai?

No caso o `x:Category` que aparece no meu vídeo é apenas uma anotação da IDE, o código é `x => x.Id` apenas!
A IDE coloca esse `Category` em cinza mais claro apenas para nos dizer que estamos trabalhando com a classe `Category`.

**Comment by oliveiramdaniel** [comment]
Bom dia, @andrebaltieri , tudo otimo!
Muito obrigado.
